26/documentation.html

```diff
@@ -30,7 +30,7 @@
     <!--#include virtual="toc.html" -->
   </div>
 	<div class="right">
-		<!--#include virtual="../includes/_docs_banner.htm" -->
+		<!--//#include virtual="../includes/_docs_banner.htm" -->
```

Comment:
@mimaison you'll want to keep the `<!--#include virtual="../includes/_docs_banner.htm" -->` link format 
cf https://github.com/apache/kafka-site/pull/307